fl_training.py

```diff
@@ -158,7 +158,7 @@ def split_in_minibatches(minibatch_count, x_train, y_train):
 
     return minibatched_x_train, minibatched_y_train
 
-def prepare_aggregation_weights(aggregation_weighting, nodes_count, node_list):
+def prepare_aggregation_weights(aggregation_weighting, nodes_count, node_list, input_weights):
```

Comment:
I gave a generic name to this argument because we might developed additional weighting approaches for which we need to pass to this method a list of input weights. So I think it is cleaner to do that way (instead of creating rigidity for a specific case like the weighting per local perf). Is this OK with you @RomainGoussault ?